pkg/controller/windowsmachine/metrics/metrics.go

```diff
@@ -97,6 +98,10 @@ func Add(ctx context.Context, cfg *rest.Config, namespace string) error {
 
 		}
 	}
+	err = updateServiceMonitors(cfg, namespace)
```

Comment:
Is there a reason this cannot be done when creating the service monitors? PS: the question implies that a comment is required to explain why.

Reply:
Did you forget to add the comment?

Reply:
Added a detailed comment [here](https://github.com/openshift/windows-machine-config-operator/pull/234/files#diff-ccf7cfc3caf950d956bbe879bfbd3fd978235b1e00781499c93510d8e95ff994R238).

Reply:
A method comment describes what the method does. You need a comment here at the call site as to why you need to call `updateServiceMonitors` soon after creating them.